codespell_lib/_codespell.py

```diff
@@ -974,6 +975,15 @@ def parse_file(
         if line.rstrip() in exclude_lines:
             continue
 
+        extra_words_to_ignore = set()
+        match = inline_ignore_regex.search(line)
+        if match:
+            extra_words_to_ignore = set(
+                filter(None, (match.group("words") or "").split(","))
+            )
+            if not extra_words_to_ignore:
+                continue
```

Comment:
Isn't this a no-op currently, and a potential bear trap if some additional line-based processing is added later after this statement, as it then won't be actioned if there aren't extra_words_to_ignore?

Reply:
That's for sure not a no-op.  It shouldn't.  Try deleting the line and tests should fail.  If at that point `extra_words_to_ignore` is empty, we have the semantics to ignore the line.  That is the wildcard ignore.  

Otherwise `extra_words_to_ignore` holds additional (extra) words which are only used for this line.  

Just the lines above we have the same or similar construct: `if line in exclude_lines: continue`. So there is also an early return.

Reply:
Apologies @kaste I was missing the `if match` statement a few lines up, failing to read it over the PR comment above this one.